app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Welcome to Viewing Party!</h1>
+
+<p>Welcome to Viewing Party! This app allows ....</p>
+
+<div class='form'>
```

Comment:
Wondering if we'll have to use a form_for tag eventually to attach the info to a session/user model.